examples/inverse/plot_custom_inverse_solver.py

```diff
@@ -97,11 +97,6 @@ def apply_solver(solver, evoked, forward, noise_cov, loose=0.2, depth=0.8):
 
     loose, forward = _check_loose_forward(loose, forward)
 
-    # put the forward solution in fixed orientation if it's not already
-    if loose == 0. and not is_fixed_orient(forward):
```

Comment:
This will never be true because the previous call to `_check_loose_forward` handles this case.

Reply:
indeed !